e2e/process-services-cloud/task-list/edit-task-filters-component.e2e.ts

```diff
@@ -116,14 +112,14 @@ describe('Edit task filters cloud', () => {
 
         await tasksCloudDemoPage.taskListCloudComponent().checkContentIsNotDisplayedByName(assignedTaskName);
         await tasksCloudDemoPage.taskListCloudComponent().checkContentIsDisplayedByName(completedTaskName);
-        await openFilter();
+        await editTaskFilter.collapseFilter();
```

Comment:
as discussed let's keep the openFilter instead of expandFilter and closeFilter instead of collapse